lighthouse-core/report/html/templates.html

```diff
@@ -324,22 +323,6 @@
       background-color: var(--color-gray-200);
       outline: none;
     }
-    .lh-tools__dropdown .report-icon {
-      cursor: pointer;
-      background-repeat: no-repeat;
-      background-position: 8px 50%;
-      background-size: 18px;
-      background-color: transparent;
-      text-indent: 18px;
-    }
-    .dark .report-icon {
-      color: var(--color-gray-900);
-      filter: invert(1);
-    }
-    .dark .lh-tools__dropdown a:hover,
-    .dark .lh-tools__dropdown a:focus {
-      background-color: #BDBDBD;
-    }
     /* copy icon needs slight adjustments to look great */
```

Comment:
this selector doesnt apply anymore fwiw.

it seemed weird so i checked it out. it seems to be shrinking the icon to not be so big.

![image](https://user-images.githubusercontent.com/39191/116475900-4e913300-a82f-11eb-9b5e-8d11a122a126.png)

IMO its not a huge deal so we can just nuke the rule. (or move it to `.report-icon--copy::before`)